dvc/stage.py

```diff
@@ -210,8 +210,11 @@ def _changed_deps(self):
         for dep in self.deps:
             if dep.changed():
```

Comment:
We could simply use dep.status() here and extract the message from it, without calling changed() first.